SAKPaaS/tslint.json

```diff
@@ -59,6 +59,7 @@
     ],
     "no-non-null-assertion": true,
     "no-redundant-jsdoc": true,
+    "no-shadowed-variable": false,
```

Comment:
Warum? Ist doch eigentlich ein ganz sinnvolles Feature.

Reply:
Ja tendenziell schon, aber mein Linter hat in der Pipe rumgemeckert:
```typescript
    this.mapService.getMapCenter().pipe(
      filter(coordinates => !!coordinates),
      filter(_ => this.mapService.getCurrentMapZoomLevel() > MapService.ZOOM_LIMIT), // HIER
      // [...]
      tap(_ => this.updateLoadingState(false)), // HIER
      // [...]
```

Er hat die `_` jeweils als shadowed-var markiert - was ja eigentlich nicht stimmt, oder? :D @timgrohmann

Reply:
Laut https://palantir.github.io/tslint/rules/no-shadowed-variable/ geht es auch so:
```json
"no-shadowed-variable": [
  true,
  {
    "underscore": false
  }
]
```
Dann würden wir das Feature nicht komplett töten ;)

Reply:
Ja good point :D
Hab halt den Fehler nicht so wirklich verstanden, weil in anderen Files kann ich auch mit der Regel `_` als Variablen verwenden. Zum Beispiel in `map.component.ts`:
```typescript
filter(_ => this.mapService.getCurrentMapZoomLevel() > MapService.ZOOM_LIMIT)
```

Naja anyway, hab die tsconfig wieder angepasst